.jscsrc

```diff
@@ -0,0 +1,5 @@
+{
+	"preset": "jquery",
+        // Since we check quotemarks already in jshint, this can be turned off
+        "validateQuoteMarks": false
```

Comment:
Is this still needed?

As far as I understand JSCS was merged with eslint, so we should need only one config file.

If that's a quick fix maybe we could do it in this PR, if not consider if for some cleanup later.

Reply:
I just realised we use jshint, not eslint - so never mind. Let's keep it as it is and consider eslint later.